### PR TITLE
Remove SIWE functionality

### PR DESCRIPTION
**Pull request by rudolfs**
It's not working with our current seed nodes and is confusing to users. https://github.com/radicle-dev/radicle-interface/issues/528
It also prevents us from upgrading transitive dependencies. https://github.com/radicle-dev/radicle-interface/issues/520

### CODE REVIEW

File: vite.config.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ export default defineConfig({
       output: {
         manualChunks: {
           ethers: ["ethers", "@ethersproject/abstract-provider"],
-          auth: ["siwe", "@walletconnect/client"],
+          auth: ["@walletconnect/client"],
```
[the remark this answers is not in the document]
Comment (rudolfs) [reply vite.config.ts:67]: It just pushes it over the edge:
```
build/assets/auth-3a7d10c8.js           84.32 kB │ gzip:  24.10 kB
```

I could move it into the ethers thing and maybe rename it to ethereum?

[the remark this answers is not in the document]
Comment (sebastinez) [reply vite.config.ts:67]: hmm yeah that should work 👍

[the remark this answers is not in the document]
Comment (rudolfs) [reply vite.config.ts]: Force pushed this change:

```diff
-          ethers: ["ethers", "@ethersproject/abstract-provider"],
-          auth: ["@walletconnect/client"],
+          ethereum: [
+            "ethers",
+            "@ethersproject/abstract-provider",
+            "@walletconnect/client",
+          ],
```

[the remark this answers is not in the document]
Comment (sebastinez) [reply vite.config.ts]: LGTM

**Pull request merged**